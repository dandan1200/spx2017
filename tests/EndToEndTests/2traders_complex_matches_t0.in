BUY 0 GPU 10 20;
BUY 1 GPU 100 30;
SELL 2 Router 5 20;
AMEND 1 100 40;
