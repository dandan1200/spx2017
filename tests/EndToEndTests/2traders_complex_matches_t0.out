MARKET OPEN;
ACCEPTED 0;
ACCEPTED 1;
MARKET SELL GPU 5 35;
MARKET BUY Router 4 22;
ACCEPTED 2;
FILL 2 4;
AMENDED 1;
FILL 1 5;
